index.html

```diff
@@ -54,40 +57,18 @@ <h2 class="support-text">I’am a Software Developer</h2>
             <h2 class="title1" >My Recent Works</h2>
             <div class="line-recent-works"></div>
         </div>
-        <div class="img-data-and-btn-seemore">
-            <div class="img-gymfit">
-                <img src="images/multi-post.png" class="multi-p-image" alt="multi">
-            </div>
-            <div class="div-informatn">
-                <div class="for-med-multipost">
-                <h4 class="multipost-h1">Multi-Post Stories</h4>
-                </div>
-                <div class="para-info">
-                <p>A daily selection of privately personalized reads;
-                    no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
-                    when an unknown printer took a standard dummy text.</p>
-                </div>
-                <div class="list-items">
-                <ul>
-                    <li>css</li>
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                </div>
-                <div class="button-see">
-                <button class="button-see-somethn" type="button">
-                    See Project
-                </button>
-                </div>
-            </div>
-            </div>
+                
+            <!-- Page dynamic Section-->
+        <div id="portfolioModal">
+        <span class="close">&times;</span>
+        <div class="portfolio-modal-content"></div>
+        </div>
+        <div id="recentProjectWorks"></div>
     </section>
-
     <section class="project-portfolio">
         <div class="mediacss-grid">
             <div class="gitem1">
-                <button class="buttongrid1" type="button">See Project</button>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
```

Comment:
- [ ] Please try to stick to Figma design as much as possible to make your work look professional .so you can try to leverage the positioning of items in the popup modal to match the design (fonts,colors, margins,padding,etc)

your design:
![image](https://user-images.githubusercontent.com/90524466/180284211-49751801-15d5-43c9-b98c-2baa8b272b09.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/180284477-b6c7e77e-9ee7-4e42-9e54-700e91fcc2d0.png)